### ISSUE TITLE
Worse performance in v0.2.2 compared to v0.2

### ISSUE THREAD

**Issue by guderian3000** [StefReck/MEdgeConv#1]
Training performance has decreased between version 0.2 and 0.2.2. For the case of a classification I attach the plots produced during training. A difference after 3 epochs of 6% in the accuracy can be observed. The Hyperparameters were the exact same for both cases.
[summary_plot-1.pdf](https://github.com/StefReck/MEdgeConv/files/5163754/summary_plot-1.pdf)
[summary_plot.pdf](https://github.com/StefReck/MEdgeConv/files/5163755/summary_plot.pdf)

**Comment by StefReck** [comment]
Looks like I can't reproduce this. Are you sure the models are identical? (You can check this e.g. via the model_arch.json)

**Comment by guderian3000** [comment]
Ah, I have an idea what happened:

With version 0.2.0 I did this scan of the activation functions:
[ORCA4_graph_activation.pdf](https://github.com/StefReck/MEdgeConv/files/5164154/ORCA4_graph_activation.pdf)
My conclusion: sigmoid actually performs best (lol). But due to the bug you fixed recently for version 0.2.2 the activation function was never changed and is indeed "relu" for every case in the plot as can be seen in the json files.

Then, with the new version the sigmoid is actually taken into account and seems to give worse results. By choosing relu again I should end up with the same performance again. I am verifying this at the moment. Also I should repeat the scan through the activation functions^^

**Comment by guderian3000** [comment]
Yeah ok, it was like that. In that case everything ok :)
